components/rules-watcher/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1alpha1
```

Comment:
Could we call the component `rule-watcher`?

I think its a good idea to avoid plural names, `rule` sufficiently describes "one or many" IMO.